src/core/stream_set.c

```diff
@@ -545,6 +545,10 @@ QuicStreamSetNewLocalStream(
         //
         Stream->OutFlowBlockedReasons |= QUIC_FLOW_BLOCKED_STREAM_ID_FLOW_CONTROL;
         Stream->BlockedTimings.StreamIdFlowControl.LastStartTimeUs = CxPlatTimeUs64();
+        QuicSendSetSendFlag(&Stream->Connection->Send,
+                            STREAM_ID_IS_UNI_DIR(Stream->ID)?QUIC_CONN_SEND_FLAG_UNI_STREAMS_BLOCKED
+                            :QUIC_CONN_SEND_FLAG_BIDI_STREAMS_BLOCKED);
```

Comment:
```suggestion
        QuicSendSetSendFlag(
            &Stream->Connection->Send,
            STREAM_ID_IS_UNI_DIR(Stream->ID) ? 
                QUIC_CONN_SEND_FLAG_UNI_STREAMS_BLOCKED : QUIC_CONN_SEND_FLAG_BIDI_STREAMS_BLOCKED);
```
Also, should we do this on line 528 above as well? I think we probably should.

Reply:
updated. 
Any tool to format the code?